src/Libraries/Microsoft.Extensions.Http.AutoClient/README.md

```diff
@@ -1 +1,229 @@
-README
+# Microsoft.Extensions.Http.AutoClient
+
+This package allows creating efficient HTTP clients for existing REST APIs with minimal code. The client code is generated at compile time, is highly efficient, and integrates with modern telemetry patterns.
+
+## Install the package
+
+```dotnetcli
+dotnet add package Microsoft.Extensions.Http.AutoClient
+```
+
+Or directly in the C# project file:
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.Extensions.Http.AutoClient" Version="[CURRENTVERSION]" />
+</ItemGroup>
+```
+
+## Usage Example
+
+### Defining a client
+
+The first step to generate a client is to define an interface that describes the client. The interface must be marked with the `[AutoClient]` attribute[^1].
```

Comment:
I'm wondering if the brackets should be removed or add ("MyHttpClient") here as well. I know there's an example below but quick readers might just see "[AutoClient]" and go with that. No strong opinion though :)